addons/account/models/account_move.py

```diff
@@ -3983,9 +3983,9 @@ def button_draft(self):
             exchange_move_ids = set([row[0] for row in self._cr.fetchall()])
 
         for move in self:
-            if move.id in exchange_move_ids:
+            if not self._context.get("force_draft") and move.id in exchange_move_ids:
```

Comment:
Hello @andreagidaltig I would use a more descriptive context like

`force_draft_in_fx_and_caba_entries`

Other than that It looks good to me.
cc/ @luistorresm 

Best Regards.

Reply:
Better explicit than implicit? :laughing: